include/RAJA/policy/sycl/forall.hpp

```diff
@@ -64,10 +66,10 @@ namespace impl
  ******************************************************************************
  */
 RAJA_INLINE
-cl::sycl::range<1> getGridDim(size_t len, size_t block_size)
+::sycl::range<1> getGridDim(size_t len, size_t block_size)
 {
   size_t size = {block_size * ((len + block_size - 1) / block_size)};
```

Comment:
I don't understand this. Why multiply by `block_size` and divide by `block_size`?

Reply:
The integer division gives us the minimum number of blocks needed to cover `len`.  We then multiple by the `block_size` to get the global size.

Reply:
We normally do the divide with RAJA_DIVIDE_CEILING_INT.